mappings/net/minecraft/world/biome/Biome.mapping

```diff
@@ -23,33 +23,13 @@ CLASS bie net/minecraft/world/biome/Biome
 		METHOD c temperature (F)Lbie$a;
 		METHOD d downfall (F)Lbie$a;
 	CLASS bie$b Category
-		FIELD a NONE Lbie$b;
-		FIELD b TAIGA Lbie$b;
-		FIELD c EXTREME_HILLS Lbie$b;
-		FIELD d JUNGLE Lbie$b;
-		FIELD e MESA Lbie$b;
-		FIELD f PLAINS Lbie$b;
-		FIELD g SAVANNA Lbie$b;
-		FIELD h ICY Lbie$b;
-		FIELD i THE_END Lbie$b;
-		FIELD j BEACH Lbie$b;
-		FIELD k FOREST Lbie$b;
-		FIELD l OCEAN Lbie$b;
-		FIELD m DESERT Lbie$b;
-		FIELD n RIVER Lbie$b;
-		FIELD o SWAMP Lbie$b;
-		FIELD p MUSHROOM Lbie$b;
-		FIELD q NETHER Lbie$b;
+		FIELD i THEEND Lbie$b;
```

Comment:
This is not a typo (by me). The constant appears like that without an underscore in the byte code. Might be a typo by Mojang, though.